test/test_get.py

```diff
@@ -10,8 +10,18 @@
 
 
 @pytest.fixture
-def downloader():
-    return Downloader(cache_dir=None)
+def downloader(request):
+    return request.getfixturevalue(request.param)
+
+
+@pytest.fixture
+def downloader_without_specified_cache_dir():
+    return Downloader()
+
+
+@pytest.fixture
+def downloader_with_specified_cache_dir():
+    return Downloader(cache_dir="./.cache")
```

Comment:
tests should not write to the project directory. for keeping a folder around for multiple tests, we usually use pytest fixture scopes (module or session scope). can get tricky though. :)